junit-platform-console/src/main/java/org/junit/platform/console/options/UriConverter.java

```diff
@@ -20,7 +20,7 @@
 /**
  * @since 1.0
  */
-class URIConverter implements ValueConverter<URI> {
+class UriConverter implements ValueConverter<URI> {
```

Comment:
Shouldn't this match the datatype name? The class you are converting from is called `java.net.URI`. Doesn't it make sense to be consistent with that?

Reply:
I think it would be better for it to be consistent with the other class `UriSelector` that uses this Capital-Letter-Then-Lowercase format, but that's probably because I'm biased and prefer Google's Java Style Guide for class naming, which advocates this style. :)

If the JUnit 5 team agree with you on this, then I'd be more than happy to make `UriSelector` consistent with this class instead (i.e. naming it to `URIConverter`).

Reply:
I think renaming `UriSelector` to `URISelector` seems like the more reasonable approach but I'm just a random contributor so it is best to wait until the team gives you feedback.

Reply:
> Shouldn't this match the datatype name? The class you are converting from is called `java.net.URI`. Doesn't it make sense to be consistent with that?

Yes, that was my original intent.

> I think renaming `UriSelector` to `URISelector` seems like the more reasonable approach but I'm just a random contributor so it is best to wait until the team gives you feedback.

That would be a breaking change now that 5.0 is released.

I think we should leave it as-is.

@junit-team/junit-lambda Thoughts?

Reply:
"leave it as-is" +1

Reply:
@marcphilipp

> That would be a breaking change now that 5.0 is released.

Oh, really? I'm sure I'm brainfarting ATM (for a lack of a better term), but I can't think of why this would be the case, as by my understanding `UriSelector` is package-private and thus inaccessible by JUnit 5 users, so changing its name shouldn't be a breaking change.

Reply:
`UriSelector` is public and marked as "stable".

Reply:
Ahhh, makes sense to me. Ta, @marcphilipp, for clarifying this for me. :)